tests/acceptance/features/apiAntivirus/serviceAvailabilityCheck.feature

```diff
@@ -0,0 +1,15 @@
+Feature: service health check
+
+
+  Scenario: health check
+    When a user requests these endpoints:
```

Comment:
we already have this step and can be used here if you are not using `service` column internally
```suggestion
    When a user requests these endpoints with "GET" and no authentication
```

Reply:
yes, I know. but I don't use `https:` and `9200` in the $this->featureContext->getBaseUrl()
so I created getBaseUrlWithoutSchemeAndPort()

Reply:
if we add `"code" => "%base_url_without_scheme_and_port%",` then it might also work with other steps. if not then we can add `substituteInLineCodes()` wherever requried.

Reply:
we need http url not https (which most of the step use)